### ISSUE TITLE
NPE on configuration request

### ISSUE THREAD

**Issue by Zelldon** [zeebe-io/zeebe#3335]
**Describe the bug**

I saw this lately in our benchmarks and now also in our QA tests (for example DeploymentClusteredTest)

```
11:01:23.549 [] [raft-server-2-raft-atomix-partition-3] DEBUG io.atomix.protocols.raft.roles.LeaderAppender - RaftServer{raft-atomix-partition-3} - Configuring 0
11:01:23.563 [] [raft-server-0-raft-atomix-partition-3] ERROR io.atomix.utils.concurrent.SingleThreadContext - An uncaught exception occurred
java.lang.NullPointerException: configuration cannot be null
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:897) ~[guava-28.1-jre.jar:?]
	at io.atomix.protocols.raft.cluster.impl.RaftClusterContext.configure(RaftClusterContext.java:589) ~[atomix-raft-3.2.0-SNAPSHOT.jar:?]
	at io.atomix.protocols.raft.cluster.impl.RaftClusterContext.reset(RaftClusterContext.java:559) ~[atomix-raft-3.2.0-SNAPSHOT.jar:?]
	at io.atomix.protocols.raft.roles.AbstractRole.updateTermAndLeader(AbstractRole.java:117) ~[atomix-raft-3.2.0-SNAPSHOT.jar:?]
	at io.atomix.protocols.raft.roles.InactiveRole.onConfigure(InactiveRole.java:76) ~[atomix-raft-3.2.0-SNAPSHOT.jar:?]
	at io.atomix.protocols.raft.impl.RaftContext.lambda$registerHandlers$13(RaftContext.java:765) ~[atomix-raft-3.2.0-SNAPSHOT.jar:?]
	at io.atomix.protocols.raft.impl.RaftContext.lambda$runOnContext$38(RaftContext.java:794) ~[atomix-raft-3.2.0-SNAPSHOT.jar:?]
	at io.atomix.utils.concurrent.SingleThreadContext$1.lambda$execute$0(SingleThreadContext.java:53) ~[atomix-utils-3.2.0-SNAPSHOT.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) [?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

But it cause not really a problem, because it is retried after 5 seconds and then it seems to work

**Comment by deepthidevaki** [comment]
I think this is same as #4137 and should be fixed by #4219

**Comment by deepthidevaki** [comment]
Sorry. This is a different issue. But duplicate of #4159
